piker/brokers/kraken.py

```diff
@@ -106,6 +119,15 @@ class Pair(BaseModel):
     ordermin: float  # minimum order volume for pair
 
 
+class Trade(BaseModel):
+    """Trade class that helps parse and validate ownTrades stream"""
```

Comment:
I've started adopting a doc string format with `'''`.

I'm not sure if it'll still but for now I think i'd prefer to give it a try.
It includes a final blank line before the closing `'''`.